README.rst

```diff
@@ -101,17 +101,37 @@ Running Tests
 Setup
 #####
 
-Tox, our test runner, tests against both Python 2.7 and Python 3.4
-environments. We have a set of tests in the :code:`dcos` package (root
-directory) and in the :code:`dcoscli` package (:code:`cli` directory). When
-running the tests describe below change directory to one of those two and
-follow the instructions.
+Tox, our test runner, tests against Python 3.4. We have a set of tests in
+the :code:`dcos` package (root directory) and in the :code:`dcoscli` package
+(:code:`cli` directory). When running the tests describe below change
+directory to one of those two and follow the instructions.
+
+
+Initialization
+#######
+
+The `config` integration tests use static config files. To run these tests
+make sure you set owner only permissions on these files:
+
+:code:`chmod 600 cli/tests/data/dcos.toml`
```

Comment:
Also see #969, which does it in Python, in case someone doesn't notice the `Makefile` and runs `tox` directly (like I did 😄 )